### PR TITLE
The Illusive Language Re-Flavor PR 

### PR DESCRIPTION
**Pull request by Draganfrukts**
## About The Pull Request

This PR focuses on re-writing a good chunk of our custom languages to better suit the universe around them and increase the quality in which they're presented.
image
ADDITIONAL CHANGES (2/7)
**Chitin-Click**
A popular non-Human language that finds extensive use by various types of anthropomorphic invertebrates the likes of Mothpeople and other types of insects. It consists of complex flutters, chittering, antenna movements, and sparse guttural syllables. In modern Sol standard writing, some of its sounds have been compared to Ancient Swedish (Dating back to the Human 19th century).

**Skrelluq**
Popularly known as \"Skrellian\" by foreigners, this newly-discovered language that the Skrell employ follows no traditional speech patterns. It relies on various differently-pitched warbles and low-frequency sound to construct different sentences and is nearly inaudible to non-Skrell and anyone lacking an appropriate implant.

**Yangyu**
Also popularly known as \"Konjin\", this language group formally regarded as Orbital Sino-Tibetan is a result of a genetic relationship between Chinese, Tibetan, Burmese, and other Human languages of similar characteristics that was first proposed in the early 19th century and is extremely popular even in the space age. Originating from Asia, this group of tongues is the second most spoken by Human and Human-derived populations since the birth of Sol Common - and was a primary contender to be the Sol Federation's official language. Many loanwords, idioms, and cultural relics of Japanese, Ryukyuan, Korean, and other societies have managed to persist within it, especially in the daily lives of speakers coming from Martian cities.

**Hearthkin**
On top of their native tongue, primitive felinids now speak Siik'Tajr over Uncommon due to their matching linguistical biology (The presence of additional feline bodyparts) and the interest that their alternative language poses (Trade and communication with outsiders)

**Plutonian**
Plutonian Franco-Castilian is a constructed Romance language that was developed early on in the Sol Federation's colonization history out of necessity for communication between its first Plutonian colonists. It heavily borrows from Spanish and French, with minor influence from other tongues the likes of Italian and Portuguese, despite coming off as elegant it carries a heavy amount of slang and idioms correlated to certain criminal groups. Today, it stands heavily engrained in the planet's culture - and almost every citizen will speak at least some of it on top of Sol.

## How This Contributes To The Skyrat Roleplay Experience

Currently, a portion of these languages are badly written and lack both substance and justification. Why the hell would people be casually speaking in pure Japanese in the 26th century? Why is there a dedicated language for poor citizens?
It also broadens the character possibilities by mechanically expanding the pool that you can choose from for cultures and tongues. You /could/ do it before this PR, but this makes things easier.

## Changelog

:cl:
spellcheck: Reflavored and/or expanded most custom languages. Hearthkin now use Siik'Tajr over Uncommon.
image: Improved the quality of several language icons. Changed the icon of Pan-Slavic to the Pan-Slavic colors instead of the NRI flag, as it represents various cultures and not a State.
/:cl:

### CODE REVIEW

**Comment by Nerev4r** [comment]
I like this. These hit a lot better than the previous versions.

**Comment by Paxilmaniac** [comment]
Chinese catgirls will become real

**Comment by Gyrandola** [comment]
I don't like that you made ALL those languages  specifically tied to a planet or a culture. You are leaving no lorefriendly choice for custom lore characters that come from outside known nations/systems and who could, at least, pick Spacer or Gutter to symbolize their "outsider" background.

Don't get me wrong, I do like how you characterized Sol etcetera but ultimately the removal of all "generic" languages hurts player freedom in a subtle way.

**Comment by tf-4** [comment]
GOOD PR

File: modular_skyrat/modules/customization/modules/language/piratespeak.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /datum/language/piratespeak
-	name = "Pirate Gutter"
-	desc = "The language of space pirates. Sounds similar to gutter, though intentionally filled with odd slang to make deciphering hard."
+	name = "Spatial Sea-talk"
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/language/piratespeak.dm]:
```suggestion
	name = "Spatial Sea-tongue"
```
Sea-talk sounds weird in my personal opinion; tongue may not be better tho so its up for debate.
Also worth considering, "Spacial" as a synonym which may fit better to emphasize solar-space vs material-space (https://thesaurus.plus/related/spacial/spatial)

File: modular_skyrat/modules/customization/modules/language/spacer.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /datum/language/spacer
 	name = "Spacer"
-	desc = "A rough, informal language used infrequently by humans as a last resort when attempts to establish dialogue in more proper languages fail and no autotranslators are available."
+	desc = "A rough, informal tongue originally used by a rogue Tajaran crew in a trading post as a last resort when attempts to establish dialogue in more proper languages fail and no automatic translators are available. It relies heavily on tone, body language, signing, and a multitude of creole loanwords. While its use has fallen severely over the years, it's still practiced by a quantity of Frontier crews and favored by the Free Trade Union."
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/language/spacer.dm]:
```suggestion
	desc = "A rough, informal tongue used as a last resort when attempts to establish dialogue in more proper languages fail and no automatic translators are available. It relies heavily on tone, body language, signing, and a multitude of creole loanwords. While its use has fallen severely over the years, it's still practiced by a quantity of Frontier crews and favored by the Free Trade Union."
```
The backstory really just makes it a confusing read, with sudden changes between tenses and implying that the language was once-used and now dead.
Informal languages like this are so vague that backstory is pretty much impossible anyways. There'd have been so many contact points with different interpretations to the translations that in current spacer history, the language wouldn't be consistent enough to really qualify as one.

**Comment by ghost** [comment]
Icon states check out, the new ones look real clean ❤️ 
Not going to debate the pan-slavic flag change since I have no say in that lore but it's worth noting the drastic change for other reviewers

**Comment by Draganfrukts** [comment]
> I don't like that you made ALL those languages specifically tied to a planet or a culture. You are leaving no lorefriendly choice for custom lore characters that come from outside known nations/systems and who could, at least, pick Spacer or Gutter to symbolize their "outsider" background.
> 
> Don't get me wrong, I do like how you characterized Sol etcetera but ultimately the removal of all "generic" languages hurts player freedom in a subtle way.

You don't need to belong to a State nor a culture to speak a specific language. IE, I don't have a drop of Anglo blood or history in me yet I speak English.
Spacer is also very much still a generic language.

File: modular_skyrat/modules/customization/modules/language/yangyu.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /datum/language/yangyu
-	name = "Spatial Sino-Tibetan"
-	desc = "A genetic relationship between Chinese, Tibetan, Burmese and other Human languages of similar characteristics was first proposed in the early 19th century and is extremely popular even in the space age. Originating from Asia, this group of tongues is the second most spoken by Human and Human-derived populations since the birth of Sol Common - and was a primary contender to be the Sol Federation's official language."
+	name = "Spacial Sino-Tibetan"
```
Comment (Nerev4r) [inline modular_skyrat/modules/customization/modules/language/yangyu.dm]: use Orbital Sino-Tibetan it goes hard
tie it into east asian orbital habitats around mars or smth

**Comment by YakumoChen** [comment]
It was mentioned with the akula language (iirc zone supported this) but descriptions need to have a more UI friendly presentation like a drop down or scroll bar to not clutter the language UI needlessly with lots of paragraphs of text.

**Comment by Draganfrukts** [comment]
> It was mentioned with the akula language (iirc zone supported this) but descriptions need to have a more UI friendly presentation like a drop down or scroll bar to not clutter the language UI needlessly with lots of paragraphs of text.

Not within the scope of this PR. In comparison to the Akulan language, they're about the right length.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Just a reminder that Xyel requested that the changes to Yangyu would be moved over to Uncommon instead, and to have Yangyu left as-is for now.

**Comment by Draganfrukts** [comment]
> Just a reminder that Xyel requested that the changes to Yangyu would be moved over to Uncommon instead, and to have Yangyu left as-is for now.

Talking this out currently

**Comment by Draganfrukts** [comment]
> Just a reminder that Xyel requested that the changes to Yangyu would be moved over to Uncommon instead, and to have Yangyu left as-is for now.

Reached an agreement and changed accordingly. It should be good to go now.

**Comment by Draganfrukts** [comment]
More discussion happening, possible changes abroad 
image

**Review by GoldenAlpharex — APPROVED** [review]
After a bunch of back and forth, I'm satisfied with where it's at now. Good work.

**Comment by GoldenAlpharex** [comment]
Haven't heard much in terms of complaints about this, so I'll go ahead and merge it.

**Pull request merged**